src/components/dashboard/components/SideNavigation.jsx

```diff
@@ -1,6 +1,14 @@
 import React from 'react';
+import { NavLink } from 'react-router-dom';
 import AddDictionary from './dictionary/AddDictionary';
-import { Link }from 'react-router-dom';
+
+const activeLinkStyles = {
+  backgroundColor: 'gray',
+  borderBottomColor: 'white',
+  color: 'white',
+  borderRadius: '10px',
```

Comment:
if you provide a CSS class as a value to the NavLink activeStyle attribute, it would still work. Inline styles are hard to find and edit especially when the codebase eventually becomes really large. I suggest you create a regular CSS class and use it.